templates/base.html

```diff
@@ -134,7 +148,37 @@
             </ul>
           </div>
         </div>
-      </li><li><a href="{{ wiki_url('Tools') }}">{{ _('Tools') }}</a></li><li><a href="{{ url('demos') }}">{{ _('Demos') }}</a></li><li><a href="{{ wiki_url('Mozilla/Connect') }}">{{ _('Connect') }}</a></li><li class="nav-search-link"><a href="{{ url('search') }}" title="Search"><i aria-hidden="true" class="icon-search"></i></a></li>{% if not is_sphinx %}<li class="main-nav-search"><form action="{{ url('search') }}" method="get" role="search">
+      </li><li><a href="{{ wiki_url('Zones') }}">{{ _('Mozilla Docs') }}<i aria-hidden="true" class="icon-caret-down"></i></a>
+
+        <div class="submenu js-submenu" id="nav-zones-submenu">
+          <div class="submenu-column">
+            <ul>
+              <li><a href="{{ wiki_url('Mozilla/Add-ons') }}">{{ _('Add-ons') }}</a></li>
+              <li><a href="{{ wiki_url('Firefox') }}">{{ _('Firefox') }}</a></li>
+              <li><a href="{{ wiki_url('Mozilla/Marketplace') }}">{{ _('Firefox Marketplace') }}</a></li>
+              <li><a href="{{ wiki_url('Firefox_OS') }}">{{ _('Firefox OS') }}</a></li>
+              <li><a href="{{ wiki_url('Persona') }}">{{ _('Persona') }}</a></li>
+            </ul>
+          </div>
+        </div>
+      </li><li><a href="{{ wiki_url('Tools') }}">{{ _('Developer Tools') }}</a></li><li><a href="{{ url('demos') }}">{{ _('Demos') }}</a></li>{% if not waffle.flag('header_feedback') %}<li><a href="{{ wiki_url('Mozilla/Connect') }}">{{ _('Connect') }}</a></li>{% endif %}{% if waffle.flag('header_feedback') %}<li><a href="{{ wiki_url('MDN/Feedback') }}">{{ _('Feedback') }}<i aria-hidden="true" class="icon-caret-down"></i></a>
```

Comment:
Can `{% endif %}{% if waffle.flag('header_feedback') %}` be combined into a single `{% else %}` block? And can the indentation be straightened up to make the if/else logic more clear?